mne/beamformer/_lcmv.py

```diff
@@ -164,24 +163,9 @@ def _apply_lcmv(data, info, tmin, forward, reg, noise_cov=None,
         # all other combinations are handled here:
         else:
             if pick_ori == 'max-power':
-                eig_vals, eig_vecs = linalg.eigh(Ck)
-
-                # Choosing the eigenvector associated with the middle
-                # eigenvalue. The middle and not the minimal eigenvalue is used
-                # because MEG is insensitive to one (radial) of the three
-                # dipole orientations and therefore the smallest eigenvalue
-                # reflects mostly noise.
-                for ii in range(3):
-                    if ii != eig_vals.argmax() and ii != eig_vals.argmin():
-                        idx_middle = ii
-
-                # TODO: The eigenvector associated with the smallest eigenvalue
-                # should probably be used when using combined EEG and MEG data
-                max_ori = eig_vecs[:, idx_middle]
-
-                Wk[:] = np.dot(max_ori, Wk)
-                Ck = np.dot(max_ori, np.dot(Ck, max_ori))
-                is_free_ori = False
+                raise ValueError('The unit-gain constraint beamformer is not '
+                                 'yet implemented with max-power orientation '
+                                 'selection.')
```

Comment:
Shouldn't it be:

The max-power orientation is not implemented with weight_norm set to None

?